doc/user-guide.md

```diff
@@ -156,7 +156,15 @@ return reconcile.Result{}, err
 return reconcile.Result{Requeue: true}, nil
 ```
 
-**// TODO:** Doc on controller-runtime client and examples on how to use it.
+You can set the `Resulte.RequeueAfter` to requeue the `Request` after a grace period as well:
```

Comment:
```suggestion
You can set the `Result.RequeueAfter` to requeue the `Request` after a grace period as well:
```

Just had to try the new github suggestion :)

Reply:
Feel free to just change it instead of accepting that change. :)

Reply:
Woah very cool!